packages/yad/yad.pacscript

```diff
@@ -0,0 +1,31 @@
+name="yad"
+version="10.1"
+url="https://github.com/v1cont/yad/archive/refs/tags/v$version.tar.gz"
+build_depends="autotools-dev intltool libgtk-3-dev"
+depends="libc6 libcairo2 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libpango-1.0-0 libpangocairo-1.0-0"
+replace="yad"
+gives="yad"
+description="Yet Another Dialog"
+hash="769e723a8f227ce4430f89fda4c79809ad46fb3c04d34ee466b51cf5e0f1c373"
+optdepends=("libwebkit2gtk-4.0-dev: for supporting HTML dialog"
+"libgtksourceview-4-dev: for enabling syntax highlighting in text-info dialog"
+"libgspell-1-dev: for support spell checking in text fields")
+maintainer="echometerain <echometer@disroot.org>"
+
+prepare() {
+	autoreconf -ivf
+	intltoolize
+}
+
+build() {
+	./configure
+	make
```

Comment:
Check if each of the `optdepends` is installed, and accordingly add the flags to enable those features in `./configure`

---

Reasoning:
**FROM THE ARCH REPO**
```bash
  ./configure \
    --prefix=/usr \
    --enable-icon-browser \
    --enable-html \
    --enable-gio \
    --enable-spell \
    --enable-sourceview
```

As you can see it needs to explicitly enable the flag to enable the feature. Since you put the optional features in `optdepends` you need to check if each of them is installed separately and accordingly enable the flags

---

```bash
dpkg-query -W (dependency name) > /dev/null 2>&1
```

Use that command to check if the dependency is installed, it evaluates to `true` and `false` accordingly